ui/static/ui/js/listing_resources.jsx

```diff
@@ -430,6 +432,7 @@ define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
     var ListingPage = React.createClass({
       render: function() {
         return <div>
```

Comment:
Of course, makes sense -- forgot we might use several of the same component on a page. Carry on!